packages/trader/src/App/Components/Form/InputField/input-field.jsx

```diff
@@ -208,6 +258,8 @@ class InputField extends React.Component {
                     min_is_disabled || (is_negative_disabled && calculateDecrementedValue() < 0) || !!is_disabled
                 }
                 decrementValue={decrementValue}
+                onLongPressEnd={onLongPressEnd}
+                is_incrementable_on_long_press={is_incrementable_on_long_press}
```

Comment:
Should we change this to `is_decrementable_on_long_press`?

Reply:
No, it is a single prop to increase/decrease. We also have a prop named `is_incrementable` which does both increase and decrease on click. Is there any neutral word to denote both increase and decrease?

Reply:
I see, the problem is increment and decrement doesn't have a neutral noun. Maybe just `has_long_press_action` would suffice? Based on the context of `onLongPressEnd`.

Reply:
I get it, but the prop `has_long_press_action` doesn't mention about the action it is going to perform considering that we pass prop in `InputField` component